Fix aclocal warnings:
/usr/share/aclocal/gtk.m4:7: warning: underquoted definition of AM_PATH_GTK
--- a/gtk.m4
+++ b/gtk.m4
@@ -6,3 +6,3 @@
 dnl
-AC_DEFUN(AM_PATH_GTK,
+AC_DEFUN([AM_PATH_GTK],
 [dnl 
